src/components/Display.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+const Display = ({ obj }) => {
+    console.log(obj);
+    const show = (obj) => {
```

Comment:
- [OPTIONAL] Kindly remove the `eslint-disable` statements used in this file. `Eslint` helps developers write clean, readable and industry-standard codes and using `eslint-disable` statements defeats this purpose. Kindly remove them. To solve the prop-types issue, you can read this [documentation](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html). 👍 😎